pkgs/by-name/ha/hatch/package.nix

```diff
@@ -24,12 +24,19 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-HW2vDVsFrdFRRaPNuGDg9DZpJd8OuYDIqA3KQRa3m9o=";
   };
 
+  patches = [
+    (replaceVars ./paths.patch {
+      uv = lib.getExe python3.pkgs.uv;
```

Comment:
because splicing dark magic, please give preference to `python3Packages` instead.